main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
+var Version = "v0.9.0+dev"
```

Comment:
`v0.9.9` is tagged, so `0.9.0+dev` is still too low, should at least be `0.9.9+dev``
Or, if we hard-code something, we could hard code the *target* and make it `1.0.0-alpha.<numcommits-since-last-tag>+<sha>`